source/magpie/operation.d

```diff
@@ -203,6 +204,72 @@ auto merge(JoinTypes type = Inner, T, U)(T df1, U df2, string lsuffix = "x_", st
     else static assert(0, "Invalid join type. Available join type: left, right, outer, inner(default)");
 }
 
+enum AggregateOP
+{
+    count = 1,
+    max = 2,
+    min = 3,
+    mean = 4,
+    median = 5
+}
+
+auto aggregate(int axis, T, Ops...)(T df, Ops ops)
+    if(isDataFrame!T)
+{
+    static if(axis)
+    {
+        DataFrame!(float, df.RowType.length) ret;
```

Comment:
I'm slightly concerned with the change of data types here, a data frame of ints shouldn't get converted to a df of floats just because I computed the max of a column or row.